BuildaGitServer/GitServerPublic.swift

```diff
@@ -41,6 +53,7 @@ public enum GitService: String {
     public func authorizeUrl() -> String {
         switch self {
         case .GitHub: return "https://github.com/login/oauth/authorize"
+        case .EnterpriseGitHub: return "https://\(hostname())/login/oauth/authorize"
```

Comment:
I think we should have an assert here as well, since we don't support GitHub Enterprise OAuth yet.